docs/alerts/alert-rules.md

```diff
@@ -0,0 +1,163 @@
+title: Alert Rules
+description: Alert rules are essential for monitoring your applications and infrastructure. They allow you to set specific conditions under which you will be notified of changes in metrics or events
```

Comment:
+ logs...

Reply:
I just added log events, if this is what you had in mind.